modules/message/html/compose.js

```diff
@@ -43,9 +46,15 @@ exports.create = function (api) {
       }
     })
 
+    var draftPerstTimeout = null
+    var draftLocation = location
```

Comment:
In patchwork, `location` is always the same (doesn't change on navigate) so the same draft appears in every box right now. I'm guessing this isn't intentional.

You should probably use something from `meta`. Either pass a key in from the `public`, `message`, and `channel` views (`location: id`), or use an existing key.

Reply:
@mmckegg thanks I'm still figuring out the overall structure. Will have a look tomorrow.

Reply:
Changed to use `${path}/${id}` to ensure uniqueness. Doesn't use the root message id so saves different drafts for different replies in threads. 

meta now always returns a `Struct`. [As its already used once](https://github.com/ssbc/patchwork/blob/master/modules/page/html/render/message.js#L32), it makes sense to always use rather than check to see whether or not to call the function.